Library/Homebrew/tap.rb

```diff
@@ -524,7 +524,7 @@ def install(quiet: false, clone_target: nil,
     Commands.rebuild_commands_completion_list
     link_completions_and_manpages
 
-    formatted_contents = contents.presence&.to_sentence&.dup&.prepend(" ")
+    formatted_contents = contents.presence&.to_sentence&.prepend(" ")
```

Comment:
`to_sentence` is careful to return mutable strings, so we don't need these `dup`s